src/main/scala/org/dia/algorithms/mcc/MCCOps.scala

```diff
@@ -216,31 +251,42 @@ object MCCOps {
       edgeList: RDD[(Int, Iterable[MCCEdge])], iteration: Int,
       buckerSize: Int,
       minGraphLength: Int,
-      sc: SparkContext): Array[(Int, Iterable[MCCEdge])] = {
+      sc: SparkContext): Array[(Int, (Iterable[MCCEdge], mutable.MutableList[mutable.HashSet[String]]))] = {
```

Comment:
1. Update comment description of the function in accordance with the changes
2. Is there any sort of way to avoid returning an array of such a complex tuple?